user_docs/en/userGuide.t2t

```diff
@@ -2858,10 +2858,6 @@ The following Braille displays are supported:
 
 These displays do not support NVDA's automatic background braille display detection functionality.
 
-If BrxCom is installed, NVDA will use BrxCom.
-BrxCom is a tool that allows keyboard input from the braille display to function independently from a screen reader.
-Keyboard input is possible with the Trio and BRAILLEX Live models.

```

Comment:
Just to make sure - since you've removed this line are you saying that braille input is supported with other devices as well?

Reply:
BrxCom is a kind of connection chain. BrxCom connects to the braille display and the screenreader connects to BrxCom.
All keyboard inputs are handled by BrxCom and not passed through to the screenreader. NVDA is able to handle the qwerty and the braille input itself. Other screenreaders also implemented this function, so BrxCom is not needed anymore.